src/util.js

```diff
@@ -118,4 +120,47 @@ function letMixin(Type) {
   return Type;
 }
 
-export { deprecate, serviceResponse, letMixin };
+/**
+ * Utility function for determining if detect-browser package successfully set
+ * itself up and determined the platform
+ */
+function canDeterminePlatform() {
+  return !!platform;
+}
+
+/**
+ * Utility function to get the name of the platform running the code. For a
+ * browser, it will be the browser's name; otherwise, it will be 'node'.
+ */
+function getPlatformName() {
+  return !!platform && platform.name;
```

Comment:
i don't understand how this line is meant to work - if `platform` is empty, won't this function just return `false` ? since `!!platform` will evaluate to false, and short-circuit the `&&`.